src/posts/how-to-use-JAWS-on-Windows

```diff
@@ -0,0 +1,47 @@
+
+## Getting Started with JAWS, (Basic commands)
+Once JAWS is installed and you have restarted your computer, you are ready to use JAWS! You can manually launch JAWS by double clicking on the JAWS icon on your desktop or selecting it from your Start menu. After running JAWS, you can use your keyboard shortcuts to interact with your computer. Below are some basic commands you can familiarize yourself with to get started using JAWS.
+- INSERT + H: This commands opens the JAWS Help system. The JAWS Help system provides comprehensive documentation on how to use JAWS effectively.
+- Arrow Keys (Basic Navigation): JAWS relies on keyboard commands for navigation. Arrow keys allow you to move between elements on the screen, such as links, heading or form fields.
+- TAB Key: Pressing TAB Key allows you to move between interactive elements in the application. 
+- H: Pressing the H key moved you between headings.
+- INSERT + DOWN ARROW: This keyboard shortcut will allow you to read the current line in the website.
+- INSERT + UP ARROW: This keyboard shortcut will allow you to read from the current position to the top of the document or application.
+- ALT + TAB: This keyboard shortcut will allow you to switch between open applications or windows.
+- WINDOWS KEY + M: This will minimize all windows, allowing you to access the desktop quickly.
+- Control + F: This key allows you to open the search feature within your browser.
+Listed here are some basic keyboard shortcuts you can use to start using JAWS right away. For a full list of shortcut commands, you can visit [JAWS website](https://doccenter.freedomscientific.com/doccenter/doccenter/rs25c51746a0cc/2011-10-04_jawsandmagicreadingcommands/JAWSKeystrokes.htm). JAWS provide many keyboard shortcuts for a variety of reasons. In the next section, we will explore some advanced features JAWS provide to their users. 
+
+## Advanced features
+JAWS is an incredible screen reader that has many advanced features you can explore. On top of providing a great screen reader experience, below are some advanced features you can try to customize JAWS.
+- INSERT + J: This command opens JAWS settings. After opening JAWS settings, you can navigate through the various options to customize JAWS. Some options include:
+-- Adjusting verbosity level to determine the amount of detail JAWS provides when reading text.
+-- Customize punctuation settings to control how JAWS pauses and pronounces words. 
+-- Voice Adjustment" section. Here, you can modify voice speed, pitch, volume, and other voice-related settings to suit your preferences.
```

Comment:
Looks like an extra `"` character snuck in.

Reply:
Thank you for catching!